src/Operations.jl

```diff
@@ -1853,6 +1851,7 @@ function gen_target_project(ctx::Context, pkg::PackageSpec, source_path::String,
     if projectfile_path(source_path; strict=true) === nothing
         # no project file, assuming this is an old REQUIRE package
         test_project.deps = copy(env.manifest[pkg.uuid].deps)
+        get!(test_project.deps, "Pkg", PkgUUID) # so that the test environment can be instantiated & precompiled
```

Comment:
Perhaps to catch the edge case where Pkg is another package..
```suggestion
        # add Pkg so that the test environment sandbox subprocesses can be precompiled
        if get!(test_project.deps, "Pkg", PkgUUID) != PkgUUID
            @warn """
            A package called Pkg is declared with a UUID that doesn't match the Pkg stdlib.
            This may cause unexpected behavior"""
        end
```